sdk/communication/communication-network-traversal/src/generated/src/models/index.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import * as coreHttp from "@azure/core-http";
+
+/**
+ * A TURN credentials response.
+ */
+export interface CommunicationTurnCredentialsResponse {
+  /**
+   * The date for which the username and credentials are not longer valid.
+   */
+  expiresOn: Date;
+  /**
+   * An array representing the credentials and the TURN server URL.
+   */
+  turnServers: CommunicationTurnServer[];
```

Comment:
I think we had talked about renaming to `IceServer` but up to you and Andrew when you want to make that change.